app/views/users/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<div>
+  <h1 class="font-bold text-4xl">Users#index</h1>
+  <p>Find me in app/views/users/index.html.erb</p>
+</div>
```

Comment:
- [ ] Great job for adding the view file for the #index action :+1: @barakadanny, can you remember you equally added the #show action for the users_controller? Let's equally add a view file for the #show action as requested in the requirement that's in ` #app/views/users/show.html.erb`